webui/react/src/hooks/useModal/Checkpoint/useModalCheckpointRegister.tsx

```diff
@@ -131,7 +92,7 @@ const useRegisterCheckpointModal = (onClose?: (checkpointUuid?: string) => void)
         modelName: selectedModelName,
       });
       if (!response) return;
-      closeModal();
+      modalClose();
```

Comment:
not really helpful but +1 for closeModal => less chars to type to filter down the list of suggestions in a linear var name matching situation

Reply:
I can't tell how much work that is but feel free to leave out.